dev-itpro/developer/devenv-data-transfer.md

```diff
@@ -138,6 +138,24 @@ begin
 end;
 ```
 
+### Example 2 - replace ModifyAll()
+
+In order to replace a Record.ModifyAll() with the DataTransfer data type, ensure that only one table is set (as both source and destination), apply filters if required, and then specify one or more new field values:
+
+```AL
+local procedure CopyFieldsReplacingModifyAll()
+var
+    dt: DataTransfer;
+    dest: Record Destination;
+begin
+    dt.SetTables(Database::Destination, Database::Destination);
+    dt.AddSourceFilter(dest.FieldNo("Field 1"), '=%1', 'A');
+    dt.AddSourceFilter(dest.FieldNo("Field 2"), '%1..%2', 'B', 'C');
+    dt.AddConstValue(dest."Enum Field"::SomeValue, dest.FieldNo("Enum Field"));
+    dt.CopyFields();
+end;
+```
+
```

Comment:
Could you include some motivation for wanting to replace ModifyAll with this? What is the positives (and negatives).

Reply:
Same answer as for your first comment, please check.